regression-tests/backends/gsql-common

```diff
@@ -40,7 +40,7 @@ gsql_master()
 		--dnsupdate=yes --resolver=8.8.8.8 --outgoing-axfr-expand-alias=yes \
 		--expand-alias=yes \
 		--cache-ttl=$cachettl --dname-processing \
-		--disable-axfr-rectify=yes $lua_prequery &
```

Comment:
I believe my tests are wrong. I shouldn't remove disable-axfr-rectify=yes there, probably a new test in regression-tests.nobackend would be more appropriate